lib/profile_component/controls/profile.rb

```diff
@@ -24,6 +44,12 @@ def self.example
         end
       end
 
+      module Initiate
+        def self.example
+          Profile.example
+        end
+      end
+
```

Comment:
A control that returns an entity shouldn't be named after a verb. Is this a profile that has been initiated? If so, Initiated makes more sense. If it's a "brand new" entity object, not having had any values assigned, call it New.